src/cascadia/TerminalSettingsModel/defaults.json

```diff
@@ -447,6 +447,7 @@
         { "command": "restoreLastClosed", "id": "Terminal.RestoreLastClosed" },
         { "command": "openAbout", "id": "Terminal.OpenAboutDialog" },
         { "command": "experimental.openTasks", "id": "Terminal.OpenTasks" },
+        { "command": "quickFix", "id": "Terminal.QuickFix" },

```

Comment:
I thought the consensus was the default action for `ctrl+shift+period` was going to be 

```
{ "command": { "action": "showSuggestions", "source": "all" }, "id": "Terminal.QuickFix" },
```
(or similar)

(with the aforementioned change to put quick fixes first, that put the quick fix immediately into the suggestions UI right at the cursor)